GitUI/UserControls/RevisionGrid/Graph/RevisionGraph.cs

```diff
@@ -576,26 +648,32 @@ public TestAccessor(RevisionGraph revisionGraph)
             }
 
             // This method will validate the topo order in brute force.
-            // Only used for unit testing.
             public bool ValidateTopoOrder()
             {
-                foreach (RevisionGraphRevision node in _revisionGraph._nodes)
+                int i = 0;
+                foreach (RevisionGraphRevision node in _revisionGraph._nodeByObjectId.Values)
                 {
-                    foreach (RevisionGraphRevision parent in node.Parents)
```

Comment:
`int i = 0` and `i++` are obsolete as well.

Reply:
Merge issue after the first `var` PR. 
Fixed  in the new rebase after the second var PR, removed the (new) line 656. i is used in printouts only.